sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -499,20 +556,22 @@ export class CertificateClient {
    *   name: "b",
    *   phone: "222222222222"
    * }]);
-   * await client.deleteCertificateContacts();
+   * await client.deleteContacts();
    * ```
    * @summary Deletes all of the certificate contacts
-   * @param options The optional parameters
+   * @param {DeleteContactsOptions} [options] The optional parameters
    */
-  public async deleteCertificateContacts(options?: RequestOptionsBase): Promise<Contacts> {
-    const span = this.createSpan("deleteCertificateContacts", options);
+  public async deleteContacts(options: DeleteContactsOptions = {}): Promise<CertificateContacts> {
+    const requestOptions = operationOptionsToRequestOptionsBase(options);
+
+    const span = this.createSpan("deleteCertificateContacts", requestOptions);
```

Comment:
The name of the span should change as well, but I am fine not doing it in this PR as lot of the recorded tests might need to change. But please log an issue to track it.